src/utils/types.js

```diff
@@ -1,8 +1,20 @@
 export default {
   colors: {
+    any: '#a0a0a0',
     image: '#92d050',
     number: '#00b0f0',
     boolean: '#ff3fcc',
     text: '#d47b00'
+  },
+  casts: {
+    image: ['any'],
+    number: ['any', 'boolean'],
+    boolean: ['any'],
+    text: ['any', 'boolean']
+  },
```

Comment:
For completeness, number and boolean values should also be castable into a text value.

Reply:
Casting numbers to bools makes sense to me, fair point. But should bools be?
I imagine something like a conditional with two text branches (possibly grouped) might be cooler. What do you think?
In that way, being able to directly cast it is certainly not _necessary_ in terms of computing power.